gcp/readme.md

```diff
@@ -0,0 +1,23 @@
+# GCP Deployment
+
+This Terraform module will deploy VMs on GCP Compute Engine.
+
+**Prerequisites:**
+- An existing subnet to deploy the VMs into. The subnet's attached firewall should allow inbound traffic on ports 22, 3000, 8888, 8889, 9090, 9092, 9644 and 33145. This module adds the `rp-node` tag to the deployed VMs, which can be used as the target tag for the firewall rule.
```

Comment:
I think ports `8888`, `8889` and currently not used

Reply:
Good catch.